src/views/SystemProfile/components/ATO/index.tsx

```diff
@@ -17,487 +17,452 @@ import {
 } from '@trussworks/react-uswds';
 import classnames from 'classnames';
 import { useFlags } from 'launchdarkly-react-client-sdk';
+import { camelCase } from 'lodash';
 
+import PageLoading from 'components/PageLoading';
 import {
   DescriptionDefinition,
   DescriptionTerm
 } from 'components/shared/DescriptionGroup';
 import Divider from 'components/shared/Divider';
 import SectionWrapper from 'components/shared/SectionWrapper';
 import Tag from 'components/shared/Tag';
+import { threatLevelGrades } from 'constants/systemProfile';
 import useCheckResponsiveScreen from 'hooks/checkMobile';
-// import { GetCedarSystems_cedarSystems as CedarSystemProps } from 'queries/types/GetCedarSystems';
+import GetSystemProfileAtoQuery from 'queries/GetSystemProfileAtoQuery';
 import {
-  tempATOProp,
-  tempCedarSystemProps
-} from 'views/Sandbox/mockSystemData';
+  GetSystemProfileAto,
+  // eslint-disable-next-line camelcase
+  GetSystemProfileAto_cedarThreat,
+  GetSystemProfileAtoVariables
+} from 'queries/types/GetSystemProfileAto';
+import {
+  SecurityFindings,
+  SystemProfileSubviewProps,
+  ThreatLevel
+} from 'types/systemProfile';
+import NotFound from 'views/NotFound';
+import {
+  formatDate,
+  showAtoExpirationDate,
+  showVal
+} from 'views/SystemProfile';
 
 import './index.scss';
 
-type ATOProps = {
-  system: tempCedarSystemProps; // TODO: Once additional CEDAR data is define, change to GQL generated type
-};
+/**
+ * Get counts of Security Findings from Cedar threat levels.
+ */
+// eslint-disable-next-line camelcase
+function getSecurityFindings(cedarThreat: GetSystemProfileAto_cedarThreat[]) {
+  return cedarThreat.reduce<SecurityFindings>(
+    (prev: SecurityFindings, curr) => {
+      const acc = prev;
+      const level = curr!.weaknessRiskLevel as ThreatLevel;
+      // Ignore nulls for non-prod environments, but should always exist on prod
+      if (level === null) return acc;
+      if (!(level in acc)) {
+        acc[level] = 0;
+      }
+      acc[level]! += 1;
+      return acc;
+    },
+    { total: cedarThreat.length }
+  );
+}
 
-const ATO = ({ system }: ATOProps) => {
+const ATO = ({ system }: SystemProfileSubviewProps) => {
   const { t } = useTranslation('systemProfile');
   const isMobile = useCheckResponsiveScreen('tablet');
   const flags = useFlags();
+  const { loading, error, data } = useQuery<
+    GetSystemProfileAto,
+    GetSystemProfileAtoVariables
+  >(GetSystemProfileAtoQuery, {
+    variables: {
+      cedarSystemId: system.id
+    }
+  });
```

Comment:
Do we need to make an individual request for Threat data (this query name is also a bit misleading), or can we include it in the `system` that's passed to this component in order to avoid an extra call to CEDAR?

Reply:
We can include everything in the parent system object if you'd like. I just laid things out this way to be more granular with data request just so the pattern might scale a bit better.

The query names were meant to reflect subcomponent names.